src/components/InformationBanner/InformationBanner.tsx

```diff
@@ -54,10 +54,8 @@ const InformationBanner: React.FC<InformationBannerProps> = ({ merknader, papirs
         return (
             <div data-testid="merknad-banner">
                 <Veilederpanel kompakt type="plakat" svg={<VeilederMaleSvg />}>
-                    <Undertittel tag="h2" className={styles.merknadBannerTitle}>
-                        Behov for mer opplysninger
-                    </Undertittel>
-                    <Normaltekst className={styles.merknadBannerParagraph}>
+                    <Undertittel tag="h2">Behov for mer opplysninger</Undertittel>
+                    <Normaltekst>
```

Comment:
Ser ut som eg har gløymt å rename alle `merknadBannerParagraph` til `paragraph` her, så det manglar styling på `<Normaltekst />`
```suggestion
                    <Normaltekst className={styles.paragraph}>
```

Reply:
Aha, det kan forklare hvorfor post-css eslint pluginnen klagde på så mange klasser :sweat_smile: